tests/trustyai/drift/utils.py

```diff
@@ -242,3 +251,34 @@ def _check_pods_ready_with_env() -> bool:
     for sample in samples:
         if sample:
             return
+
+
+def verify_metric_request(
+    client: DynamicClient, trustyai_service: TrustyAIService, token: str, metric_name: str, json_data: Any
+) -> None:
+    """
+    Sends a metric request to a TrustyAIService and validates the response.
+
+    Args:
+        client (DynamicClient): The client instance for interacting with the cluster.
+        trustyai_service (TrustyAIService): The TrustyAI service instance to interact with.
+        token (str): Authentication token for the service.
+        metric_name (str): Name of the metric to request.
+        json_data (Any): JSON payload for the metric request.
+    """
+
+    response = TrustyAIServiceRequestHandler(token=token, service=trustyai_service, client=client).send_drift_request(
+        metric_name=metric_name, json=json_data
+    )
+    LOGGER.info(msg=f"TrustyAI metric request response: {json.dumps(json.loads(response.text), indent=2)}")
```

Comment:
do we need this log as info or only as error when failing?

Reply:
I think it's helpful as info, it's not a huge payload and it happens only one, so there won't be log spamming

Reply:
helpful even if the test passes?

Reply:
Yes, I think we want to see this even if the test passes